cartographer/cloud/internal/map_builder_context_impl.h

```diff
@@ -40,14 +40,19 @@ template <class SubmapType>
 mapping::TrajectoryBuilderInterface::LocalSlamResultCallback
 MapBuilderContext<SubmapType>::GetLocalSlamResultCallbackForSubscriptions() {
   MapBuilderServer* map_builder_server = map_builder_server_;
```

Comment:
you can probably drop this line when you capture `this` anyways

Reply:
done.